docs/chaosiq-on-docker/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,47 @@
+It's recommended to use the  ```chaosiq/chaostoolkit``` Docker image from Docker Hub, which is
+provided by ChaosIQ. However if you want to build a customer Docker image, so you can add your own extensions
+the following can be used as a starter for your own Dockerfile:
+
+```
+FROM python:3.8-alpine
+
+LABEL maintainer="ChaosIQ <contact@chaosiq.io>"
+
+RUN apk add --no-cache --virtual build-deps gcc g++ git libffi-dev linux-headers \
+        python3-dev musl-dev && \
+    pip install --no-cache-dir -q -U pip && \
+    pip install --no-cache-dir chaostoolkit && \
+    pip install --no-cache-dir chaosiq-cloud && \
+    apk del build-deps
+
+ADD . /home/svc
+
+
+
+WORKDIR /home/svc
+
+ENTRYPOINT ["/usr/local/bin/chaos"]
+```
+
+Add the above to a ```Dockerfile```, it can the be built with:
+
+```
+docker build -t chaosiq .
```

Comment:
please have a look at this doc on ctk https://docs.chaostoolkit.org/deployment/k8s/operator/#run-the-experiment-with-specific-extensions

Consider indicating a section for creating a docker image with their own extensions. it also add more value to re-build its own image

Reply:
do we consider using another image name for user-custom images ? like in the link

Reply:
@dmartin35 - are you suggesting here I should use:
```
FROM chaosiq:chaostoolkit

RUN apk update && \
    apk add --virtual build-deps libffi-dev openssl-dev gcc python3-dev musl-dev && \
    pip install --no-cache-dir chaostoolkit-kubernetes && \
    apk del build-deps && \
    rm -rf /tmp/* /root/.cache
```
and then build it with 

```
docker build -t mychaosiq .
```

Reply:
- do we want to use the chaosiq ctk as base image rather than the open source one ? 
- for the docker image name, I would keep the same pattern:  my/chaostoolkit  or my/chaosiq (this is common on docker images, to have namespace/name pattern)
btw, it feels strange to make them name their docker image after our company name, wdyt ? 
- I think we could show the user how to check its extensions are well installed in their docker image, as a run custom image example: 
`docker run my/chaostoolkit info extensions`

Reply:
Really valid points David!